### ISSUE TITLE
It would be helpful if @JsonIgnoreProperties worked when writing 'any' properties

### ISSUE THREAD

**Issue by benson-basis** [FasterXML/jackson-databind#1281]
Currently, it does not. It applies when reading, but not writing, in this case.

I would settle gladly for annotating the any-getter method rather than the class for this case.

**Comment by cowtowncoder** [comment]
Right, I was wondering about this as well. At first I thought it might be confusing (wrt annotation other properties), but come to think of perhaps it would make sense. Sort of depends on whether set of any-properties is considered to be part of POJO that contains them, or more like equivalent of "unwrapped" nested object.

**Comment by cowtowncoder** [comment]
Note: the "other part" would be #1142.

**Comment by cowtowncoder** [comment]
What does work in 3.1.0 is method annotation:

```
  class MyBean {
      @JsonIgnoreProperties("secret")
      @JsonAnyGetter
      public Map<String, Object> getExtras() { ... }
  }
```

**Comment by cowtowncoder** [comment]
Adding test, marking as fixed in 3.2.0, but alreadily likely worked in 3.1.0.

Closing as fixed
